### PR TITLE
Sn 4993 did m pack

### PR DESCRIPTION
**Pull request by tmcd249**
Adds did for future mPack or wildcard messages.

### CODE REVIEW

**Review by kylemallory — CHANGES_REQUESTED** [review]
Good start! But lets drop the 'IS' from the code/classes/types/macros.

File: src/data_sets.h [hunk 1]
```diff
@@ -4609,6 +4614,32 @@ typedef struct
...
 } port_monitor_t;
 
+#define ISEvent_MAX_SIZE 64
+
+enum ISEventProtocol// make this uint8!!!
+{
+    ISEventProtocol_byte    = 1,
+    ISEventProtocol_ASCII   = 2,
+
+    ISEventProtocol_EPH     = 250,
```
[the remark this answers is not in the document]
Comment (tmcd249) [reply src/data_sets.h:4624]: it was random and really should be removed for this PR

File: src/data_sets.h [hunk 2]
```diff
@@ -4609,6 +4614,32 @@ typedef struct
...
+{
+    /** Serial number */
+    uint32_t        senderSN;
+    uint32_t        res32;
+ 
+    /** Hardware: 0=Host, 1=uINS, 2=EVB, 3=IMX, 4=GPX (see eDevInfoHardware) */
+	uint16_t        senderHdwrType;
+    uint16_t        res16;
+
+    uint8_t         protocol;
```
[the remark this answers is not in the document]
Comment (tmcd249) [reply src/data_sets.h:4637]: just didnt see it getting that large but if we think it should be larger it can be

**Review by kylemallory — APPROVED** [review]
Looks great!

**Review by waltjohnson — APPROVED** [review]

**Pull request merged**